File: Python/Vehiculo.py
class Vehiculo:
    def __init__(self, marca, color,  tipo, descripcion):
        self.marca = marca
        self.color = color
        self.tipo = tipo
        self.descripcion = descripcion
    

    # Metodos Getters 
    def get_marca(self):
        return self.marca
    
    def get_color(self):
        return self.color
    
    def get_tipo(self):
        return self.tipo
    
    def get_descripcion(self):
        return self.descripcion
    
    # Metodos Setters

    def set_marca(self,marca):
        self.marca = marca
    
    def set_color(self, color):
        self.color= color

    def set_tipo(self, tipo):
        self.tipo = tipo
    
    def __str__(self):
        return(f"Vehiculo(Marca= '{self.marca}', "
               f"Color='{self.color}', Tipo= '{self.tipo}', "
               f"Descripcion= '{self.descripcion}')")
        
    

File: Java/Vehiculo.java
public class Vehiculo {
    private String marca;
    private String color;
    private String tipo;
    private String descripcion;

    public Vehiculo(String marca, String color, String tipo, String descripcion){
        this.marca = marca;
        this.color = color;
        this.tipo = tipo;
        this.descripcion = descripcion;
    }

    public String getMarca() {
        return this.marca;
    }

    public void setMarca(String marca) {
        this.marca = marca;
    }

    public String getColor() {
        return this.color;
    }

    public void setColor(String color) {
        this.color = color;
    }

    public String getTipo() {
        return this.tipo;
    }

    public void setTipo(String tipo) {
        this.tipo = tipo;
    }

    public String getDescripcion() {
        return this.descripcion;
    }

    public void setDescripcion(String descripcion) {
        this.descripcion = descripcion;
    }

    @Override
    public String toString() {
        return "Vehiculo [marca=" + marca + ", color=" + color + ", tipo=" + tipo + ", descripcion=" + descripcion
                + "]";
    }

    
    
    
}
